sample/TimerTrigger-CSharp/function.json

```diff
@@ -5,12 +5,6 @@
       "schedule": "0 * * * * *",
       "runOnStartup": true,
       "direction": "in"
-    },
```

Comment:
This sample is causing failures - it writes queue messages to the same queue that a different CSharp sample in this same project is attempting to read as json which blows up.